Source/Evolve/WarpXEvolve.cpp

```diff
@@ -494,7 +504,48 @@ WarpX::OneStep_nosub (Real cur_time)
     ExecutePythonCallback("particlescraper");
     ExecutePythonCallback("beforedeposition");
 
-    PushParticlesandDeposit(cur_time);
+    // with collisions placed in the middle of the position push and after the momentum push
+    if (m_collisions_split_position_push) {
+        // push particles (half position and full momentum)
+        PushParticlesandDeposit(
+            a_cur_time,
+            /*skip_current=*/true,
```

Comment:
Reflect new name after #6303:
```suggestion
            /*skip_deposition=*/true,
```

Reply:
Resolved in [df7ffd7](https://github.com/BLAST-WarpX/warpx/pull/6275/commits/df7ffd784f989117779769f265dba79df38690d4).